Suomalaiset käännökset/events.json

```diff
@@ -334,7 +334,7 @@
   "Lab": {
     "NAME": "Labra",
     "DESCRIPTIONS": [
-      "Olet eksynyt huoneeseen, joka on täynnä telineitä erinäisille koeputkille, dekantterilaseja, pulloja, pinsettejä, letkupuristimia, sekoitussauvoja, pihtejä, suojalaseja, suppiloita, pipettejä, sylintereitä, lauhduttimia ja jopa harvinaisempi spiraalinen lasiputki. NL NL Miksi tiedät kaikkien näiden työkalujen nimet? Ei sillä väliä, katselet ympärillesi löytyykö mitään."
+      "Olet eksynyt huoneeseen, joka on täynnä telineittäin erinäisiä koeputkia, dekantterilaseja, pulloja, pinsettejä, letkupuristimia, sekoitussauvoja, pihtejä, suojalaseja, suppiloita, pipettejä, sylintereitä, lauhduttimia ja jopa jokunen spiraalinen lasiputki. NL NL Miksi tiedät kaikkien näiden työkalujen nimet? Ei sillä väliä, katselet ympärillesi löytyykö mitään."
```

Comment:
"Telineittäin"? Miksi adverbi muoto?

Reply:
Tulkitsin tuon niin, että huoneessa on paljon telineitä, joissa on kaikkia lueteltuja labravälineitä (eikä niin että vain koeputket ovat telineissä). Eli tavaraa on telineittäin/telinekaupalla. Nyt kun suomenkielisen lauseen lukee, niin se ei oikein osu yhteen tuon rakenteen "huone on täynnä" kanssa. Olisiko "telinekaupalla" parempi?

Reply:
Se on yksi vaihtoehto, tai sitten vaihtaa muotoon "erinäisiä koeputkitelineitä".